mod/cli/pkg/commands/jwt/jwt.go

```diff
@@ -117,16 +117,14 @@ func getFilePath(cmd *cobra.Command, path string) (string, error) {
 	// If no path is specified, try to get the cosmos client context and use
 	// the configured home directory to write the secret to the default file
 	// name.
-	if specifiedFilePath == "" {
-		clientCtx, ok := cmd.Context().
-			Value(client.ClientContextKey).(*client.Context)
-		if !ok {
-			return "", ErrNoClientCtx
-		}
-		specifiedFilePath = filepath.Join(
-			clientCtx.HomeDir+"/config/", DefaultSecretFileName,
-		)
+	clientCtx, ok := cmd.Context().
+		Value(client.ClientContextKey).(*client.Context)
+	if !ok {
+		return "", ErrNoClientCtx
 	}
+	specifiedFilePath = filepath.Join(
+		clientCtx.HomeDir+"/config/", DefaultSecretFileName,
```

Comment:
nit: filepath.Join appends the backslashes correctly according to os spec so hardcoding /config/ isn't necessary if i remember correctly